generator/repository/index.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const path = require('path');
+
+module.exports = {
+  description: 'Add reducer',
+  prompts: [
+  {
+    type: 'input',
+    name: 'name',
+    message: 'What should it be called?',
+    validate: (value) => {
+      if ((/.+/).test(value)) { return true; }
+      return 'name is required';
+    }
+  },
+  {
+    type: 'confirm',
+    name: 'wantAction',
+    default: true,
+    message: 'Do you want to add an action?'
+  }],
+  actions: (data) => {
+    let actions = [];
+    const reducerTemplate = path.resolve(__dirname, 'reducer.js.hbs');
+
+    actions.push({
+      type: 'add',
+      path: path.resolve(process.cwd(), 'src/reducers/{{camelCase name}}Reducer.js'),
```

Comment:
Any reason why should we call it reducer?

Reply:
Copy paste error it seems :(

Reply:
Resolved